SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -251,6 +273,64 @@ public void run() {
         return (T) this;
     }
 
+    private void processIAMMessage(SwrveMessage message, String eventType, Bundle bundle) {
```

Comment:
That is the whole point, to not use the singleton instance but rather communicate with the instance that launched the IAM. This is the Android way rather than using static instances.